HW7/dmuran/task_2.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+
+def check_valid_pass(password):
+    """
+    Input: password
+    Type: string
+    Return: is valid password
+    """
+    while True:
+        if len(password) < 6:
```

Comment:
if len(password) < 6 or len(password) > 16:
            return 'Not a Valid Password'

if you combine the two conditions into one it will be clearer